fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ComposeTabDrawerRobot.kt

```diff
@@ -0,0 +1,567 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@file:Suppress("TooManyFunctions")
+
+package org.mozilla.fenix.ui.robots
+
+import androidx.compose.ui.test.assertIsNotSelected
+import androidx.compose.ui.test.assertIsSelected
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.HomeActivityComposeTestRule
+import org.mozilla.fenix.helpers.TestAssetHelper.waitingTime
+import org.mozilla.fenix.helpers.TestHelper.mDevice
+import org.mozilla.fenix.helpers.click
+import org.mozilla.fenix.tabstray.TabsTrayTestTag
+
+/**
+ * Implementation of Robot Pattern for the home screen menu.
+ */
+class ComposeTabDrawerRobot(private val composeTestRule: HomeActivityComposeTestRule) {
+
+//    fun verifyBrowserTabsTrayURL(url: String) {
+//        mDevice.waitNotNull(
+//            Until.findObject(By.res("$packageName:id/mozac_browser_tabstray_url")),
+//            waitingTime,
+//        )
+//        onView(withId(R.id.mozac_browser_tabstray_url))
+//            .check(matches(withText(containsString(url))))
+//    }
+//
+//    fun verifyNormalBrowsingButtonIsDisplayed() = assertNormalBrowsingButton()
+    fun verifyNormalBrowsingButtonIsSelected(isSelected: Boolean = true) =
+        assertNormalBrowsingButtonIsSelected(isSelected)
```

Comment:
We actually planned to do some cleanup in our robots and remove this kind of duplication. So, without expecting you to do it, only if it helps you, you can skip the extra "assert" method and add the implementation directly into these "verify" methods.

Reply:
Oh nice! Yeah, I'm down to do the clean-up in this file.